### ISSUE TITLE
compilerconfig.json - relativeUrls: false does not swith off the feature

### ISSUE THREAD

**Issue by art-ist** [madskristensen/WebCompiler#140]
````json
 {
    "outputFile": "App/themes/todo/theme.css",
    "inputFile": "App/themes/todo/theme.less",
    "minify": {
      "enabled": true
    },
    "options": {
      "relativeUrls": false,
      "rootPath": "/"
    }
  }, ...
````
would not switch off relative path substitution. I couldn't rally predict the results as changing ../../ to ../ changed the resulting path dramatically.
If this really is the intended behaviour, a detailed documentation of the config file would be needed.

... THX a lot for your efforts in porviding such valuable tools for web development!

**Comment by madskristensen** [comment]
I'm not confident this is the intended behavior. The LESS compiler has a flag that will adjust relative paths of `@import` files but it won't change the path of e.g. background images from non-imported files. I've written my own relative path adjuster for that

**Comment by madskristensen** [comment]
Is this still an issue?

**Comment by riddla** [comment]
Yes, I'm experiencing this problem when compiling SASS files. Even with `"relativeUrls": false` the relative paths are adapted :/

**Comment by mlhDevelopment** [comment]
I'm using v1.11.315 on .less files and it does appear to be working as I'd expect.

**Comment by mattbrooks2010** [comment]
This still appears to be an issue with the `sass` compiler when `"relativeUrls": false` is specified in `compilerconfig.json.defaults`.

**Comment by ChristopherTotty** [comment]
Seeing relativeUrls: false not working from compilerconfig.json.defaults, but working if each file individually has the option set in compilerconfig.json.
